serializers/src/glTF/2.0/glTFAnimation.ts

```diff
@@ -241,13 +240,13 @@ export class _GLTFAnimation {
                 for (let targetAnimation of animationGroup.targetedAnimations) {
                     let target = targetAnimation.target;
                     let animation = targetAnimation.animation;
-                    if (target instanceof Mesh || target.length === 1 && target[0] instanceof Mesh) { // TODO: Update to support bones
+                    if (target instanceof TransformNode || target.length === 1 && target[0] instanceof TransformNode) {

```

Comment:
@sebavan , I'm a bit confused about the second condition we check for, why do we want to check target.length?

Reply:
because targetedAnimations can contain a target or an array of targets